question.html

```diff
@@ -19,35 +19,42 @@
 	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
 
 	<!-- need this for comlted ones. TODO: can I do this in bootstrap -->
-	<link rel="stylesheet" href="main.css">
+	<link rel="stylesheet" href="css/main.css">
 
 	<!-- Custom fonts from Google Web Font -->
-	<link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+	<link href='https://fonts.googleapis.com/css?family=Sigmar+One|Bubblegum+Sans' rel='stylesheet' type='text/css'>
 	<link href='https://fonts.googleapis.com/css?family=Montserrat:400,700|Lobster|Merriweather+Sans:400,700italic' rel='stylesheet' type='text/css'>
 
 </head>
 <body ng-controller="TodoCtrl">
 
-	<style>
-		h1{
-			font-family: 'Merriweather Sans', sans-serif;
-			font-style: italic;
-			font-size: 50px;
-		}
-		small{
-			font-family: 'Merriweather Sans', sans-serif;
-			font-style: italic;
-		}
-		a{
-			font-family: 'Merriweather Sans', sans-serif;
-			font-style: italic;
-		}
-		button{
-			font-family: 'Merriweather Sans', sans-serif;
-			font-style: italic;
-		}
-	</style>
-	<h1 size="10">Questions for {{roomId}}?</h1>&nbsp;
+	<!-- Fixed navbar -->
+	<nav class="navbar navbar-default navbar-fixed-top">
+		<div class="container">
+			<div class="navbar-header">
+				<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
```

Comment:
This does not seem to be working properly yet. 
Pushing the "collapse"-button results in no action. See image:

![bildschirmfoto 2015-10-31 um 22 15 59](https://cloud.githubusercontent.com/assets/1174752/10864117/f3b5a19e-801d-11e5-99cd-9b80cd2988a3.png)

Either leave this as a TODO and create an Issue or fix this in a new commit.